src/addons/Image/Image.js

```diff
@@ -0,0 +1,24 @@
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+
+export default class Image extends Component {
+  static propTypes = {
+    className: PropTypes.string,
+    src: PropTypes.string,
+  };
+
+  render() {
+    let classes = classNames(
+      'sd-image',
+      'ui',
+      this.props.className,
+      'image',
+    );
+    return (
+      <div {...this.props} className={classes}>
```

Comment:
The `this.props` being spread here still contains `src`.  That should be deleted before spreading.  Make a clone of the props and delete the `src` key, then spread the clone:

```jsx
const props = _.clone(this.props);
delete props.src;

<div {...props} ...
```

`alt` should be added to the delete list after adding it to the `img` as well, see next comment.